gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/FakeStorageRpc.java

```diff
@@ -91,8 +89,54 @@ public StorageObject create(StorageObject object, InputStream content, Map<Optio
   @Override
   public Tuple<String, Iterable<StorageObject>> list(String bucket, Map<Option, ?> options)
       throws StorageException {
-    potentiallyThrow(options);
-    return null;
+    String preprefix = "";
+    for (Map.Entry<Option, ?> e : options.entrySet()) {
+      if (e.getKey() == StorageRpc.Option.PREFIX) {
```

Comment:
This will probably become a case-switch as soon as we add support for other options, right?